types/react-redux/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-redux 6.0
+// Type definitions for react-redux 7.0
```

Comment:
Why was this a version bump? There's no react-redux 7.0. These are types for react-redux 6.0.

Reply:
Says in the comment further down the file as below:

```
// NOTE about the wrong react-redux version in the header comment:
// The actual react-redux version is not 6.0.0, but we had to increase the major version
// to update this type definitions for redux@4.x from redux@3.x.
// https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25321
```

Reply:
@Kovensky what would you advise is the right course of action? Should we unpublish?

Reply:
@Kovensky Simply building on the reasoning from the previous version, but if we find a suitable solution I agree that the better approach would be to follow the library targeted.